explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -122,6 +125,38 @@ export function UpgradeableProgramSection({
             {programData.authority !== null ? "Yes" : "No"}
           </td>
         </tr>
+        <tr>
+          <td>Anchor Verified</td>
+          <td className="text-lg-end">
+            <VerifiedBadge
+              loading={loading}
+              verifiedBuild={verifiedBuild}
+              programDeploySlot={programData.slot}
+            />
+          </td>
+        </tr>
+        {!verifiedBuild && (
+          <tr>
+            <td>Last Verified Deployment Slot</td>
+            <td className="text-lg-end">Never</td>
+          </tr>
+        )}
+        {verifiedBuild && verifiedBuild.verified_slot !== programData.slot && (
+          <tr>
+            <td>Last Verified Deployment Slot</td>
+            <td className="text-lg-end">
+              <Slot slot={verifiedBuild.verified_slot} link />
+              {" - "}
+              <a
+                href={buildLinkPath(verifiedBuild)}
+                target="_blank"
+                rel="noreferrer"
+              >
+                See more
+              </a>
```

Comment:
nit: can you pull this out into a `lastVerifiedState` variable which is just a react element? And then always show the "Last Verified Deployment Slot" row? I think it's fine to show that row even if the verified slot matches the program data slot